modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -10,6 +10,9 @@ import 'rxjs/add/operator/map';
 import * as models                                           from '../model/models';
 import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
 import { Configuration }                                     from '../configuration';
+{{#withInterfaces}}
+import { I{{classname}} }                                    from './{{classname}}Interface';
```

Comment:
Since the interface name convention depends on each project policy (MS is saying ["Feel free to adopt them for your own team."](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)), how about providing an option to specify the prefix/postfix word? Example, default is "I" prefix (like `IUserApi`) but if user specify "Interfece" postfix, the interface name would be `UserApiInterface` as well as interface file name. What do you think?